packages/terra-form/docs/components/Field.md

```diff
@@ -0,0 +1,23 @@
+# Field
+
+Generic form field component which handles the layout of a standard form field including errors, help text, label, value and widget placement.
+
+## Usage
+
+```jsx
+import React from 'react';
+import Field from 'terra-form/lib/Fieldset';
```

Comment:
Do you anticipate having any additional folders at the lib level? I lean towards agreeing with @windse7en